trace-api/trace-api.gradle

```diff
@@ -1,3 +1,7 @@
+plugins {
+  id 'maven-publish'
```

Comment:
Can we avoid publishing this jar for now?

Reply:
`trace-api` is now completely removed!